catalystwan/api/real_time_monitoring_api.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Cisco Systems, Inc. and its affiliates
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List
+
+if TYPE_CHECKING:
+    from catalystwan.session import ManagerSession
+
+
+class RealTimeMonitoringAPI:
+    """Real Time Monitoring API gathers information from Real time Monitoring page.
+
+    Attributes:
+        session (ManagerSession): logged in API client session
+
+    Usage example:
+        # Create session
+        session = create_manager_session(...)
+        # Get information about tenant status
+        tenant_status = session.api.dashboard.get_tenant_status()
+    """
+
+    def __init__(self, session: ManagerSession):
+        self.session = session
+
+    def get_bfd_summary(self, device_id) -> List:
+        """
+        Get information about bfd summary from a device.
+
+        Returns:
+            List of bfd summary
+        """
+        bfd_summary = self.session.get_data("/dataservice/device/bfd/summary?deviceId={}".format(device_id))
```

Comment:
This is deprecated way of adding new API.

First things is endpoint call -> all endpoints should be placed in (https://github.com/cisco-open/cisco-catalyst-wan-sdk/tree/main/catalystwan/endpoints)[endpoints directory].

Please see this PR -> https://github.com/cisco-open/cisco-catalyst-wan-sdk/pull/31/files -> it is very good representation how endpoints should be added. Look also at other endpoints examples - they will show you how to properly pass device id. Also, prefer `f'strings` instead of `.format` formatting.

Then you can use your endpoints in API, example -> https://github.com/cisco-open/cisco-catalyst-wan-sdk/blob/main/catalystwan/api/config_group_api.py

I know that it might be misleading because there are apis (like admin tech) that use raw endpoints inside - but these are obsolete and will be replaced. All new APIs should follow layered architecture.